modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/paths/CP2ActivationKeys.path

```diff
@@ -115,6 +115,49 @@
 	<td>//tbody//*[contains(., '${key_entitlementName}')]/following-sibling::*//*[contains(@class, 'btn')]</td>
 	<td></td>
 </tr>
+
+<!--ACTIVATION_KEY_DETAILS_MODAL -->
+
+<tr>
+	<td>HOST_NAME_VALUE</td>
+	<td>//div[contains(@class, 'row')][5]//*[contains(@class,'col-4')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>IP_ADDRESS_VALUE</td>
+	<td>//div[contains(@class, 'row')][7]//*[contains(@class,'col-4')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>KEY_DETAILS_MODAL</td>
+	<td>//div[contains(@class, 'modal-content') and contains(.,'Activation Key Details') and contains (.,'${key_environmentName}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>KEY_DETAILS_ELEMENT_VALUES</td>
+	<td>//div[contains(@class, 'modal-content')]//div[contains(.,'${key_group}')]//*[contains(.,'${key_label}') and contains(.,'${key_value}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>MAC_ADDRESS_VALUE</td>
+	<td>//div[contains(@class, 'row')][9]//*[contains(@class,'col-4')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>COPY_ICON_HOST_NAME</td>
+	<td>//div[contains(@class, 'row')][5]//*[contains(@class, 'lexicon-icon-copy')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>COPY_ICON_IP_ADDRESS</td>
+	<td>//div[contains(@class, 'row')][7]//*[contains(@class, 'lexicon-icon-copy')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>COPY_ICON_MAC_ADDRESS</td>
+	<td>//div[contains(@class, 'row')][9]//*[contains(@class, 'lexicon-icon-copy')]</td>
+	<td></td>
+</tr>
```

Comment:
Same as comment above

Reply:
@barrosdennys I was able to address all your comments except these paths for the copy icon. I was not able to find a way to differentiate without using the specific indexes. @MathausC would you be able to help me with my paths here to possibly avoid using index?